server/candlepin.spec.tmpl

```diff
@@ -43,8 +43,6 @@ Requires: wget
 Requires: tomcat >= 0:7.0.0
 Requires: liquibase >= 0:3.0.0
```

Comment:
Since we can remove the liquibase dependency with this PR (yay! :smile:), please replace the `Requires: liquibase...` line with a `Requires: javapackages-tools` line (it provides `/usr/share/java-utils/java-functions` and should get pulled in transitively by tomcat, but it's better to be explicit about our dependency).

Reply:
Done.